app/src/main/java/org/connectbot/TerminalView.java

```diff
@@ -323,13 +326,29 @@ public boolean deleteSurroundingText (int leftLength, int rightLength) {
 	}
 
 	@Override
+	@TargetApi(12)
 	public boolean onGenericMotionEvent(MotionEvent event) {
+		if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.HONEYCOMB) return false;
```

Comment:
I don't think it's possible to get here since this didn't exist in the parent class until API level 12 (honeycomb MR1)